spec/services/sign_in/token_serializer_spec.rb

```diff
@@ -116,16 +116,19 @@
         end
       end
 
-      context 'and client is not configured to check for anti csrf' do
-        let(:anti_csrf) { false }
+      context 'when client_config has refresh_token_path' do
+        let(:client_config) { create(:client_config, authentication:, refresh_token_path: refresh_path) }
```

Comment:
you should use the `client_config` that gets created at the begging of the test. modify it to include `refresh_token_path` then in your tests you override `refresh_token_path` instead of the entire `client_config`